amazon-kinesis-client/src/test/java/software/amazon/kinesis/lifecycle/BlockOnParentShardTaskTest.java

```diff
@@ -107,6 +109,50 @@ public final void testCallWhenParentsHaveFinished()
         assertNull(result.getException());
     }
 
+    /**
+     * Test call() when there are 1-2 parent shards that have been fully processed.
+     * @throws ProvisionedThroughputException
+     * @throws InvalidStateException
+     * @throws DependencyException
+     */
+    @Test
+    public final void testCallWhenParentsHaveFinishedMultiStream()
```

Comment:
can you add explanation of what we are trying to verify on the test cases. if the leases are marked with shard end and invoked BlockedOnParentShardTask()

Reply:
Changed.